kubernetes-manifests/frontend.yaml

```diff
@@ -66,6 +66,18 @@ spec:
             configMapKeyRef:
               name: demo-data-config
               key: DEMO_LOGIN_PASSWORD
+        - name: REGISTERED_OAUTH_CLIENT_ID
```

Comment:
You can remove the diff from `kubernetes-manifest`, it's going to happen once I release a new version of Bank of Anthos (this directory is essentially "latest release")

Reply:
Done